File: 2161-partition-array-according-to-given-pivot/2161-partition-array-according-to-given-pivot.java
class Solution {
    public int[] pivotArray(int[] nums, int pivot) {
        int[] arr=new int[nums.length];
        int a=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]<pivot){
                arr[a]=nums[i];
                a++;
            }
        }
        for(int j=0;j<nums.length;j++){
            if(nums[j]==pivot){
                arr[a]=nums[j];
                a++;
            }
        }
        for(int k=0;k<nums.length;k++){
            if(nums[k]>pivot){
                arr[a]=nums[k];
                a++;
            }
        }
        return arr;
    }
}

File: 2161-partition-array-according-to-given-pivot/2161-partition-array-according-to-given-pivot.py
class Solution:
    def pivotArray(self, nums: List[int], pivot: int) -> List[int]:
        arr=[]
        a=0
        for i in nums:
            if i<pivot:
                arr.append(i)
        for j in nums:
            if j==pivot:
                arr.append(j)
        for k in nums:
            if k>pivot:
                arr.append(k)
        return arr